From 7b9beddfd25d98dabc95f2f9bebddc555376f8ae Mon Sep 17 00:00:00 2001
From: Dmitry Artamonow <mad_soft@inbox.ru>
Date: Wed, 21 Jan 2009 21:43:30 +0300
Subject: [PATCH] SA1100: enable second serial port on h3600

It's used for communication with Atmel microcontroller (buttons,
touchscreen, etc.) We need userspace access to it for inputattach.

THIS IS A HACK!
we need to rework h3600_ts_input into proper kernel-only driver,
so we won't need inputattach.

Signed-off-by: Dmitry Artamonow <mad_soft@inbox.ru>
---
 arch/arm/mach-sa1100/h3600.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-sa1100/h3600.c b/arch/arm/mach-sa1100/h3600.c
index 3d31397..cca5223 100644
--- a/arch/arm/mach-sa1100/h3600.c
+++ b/arch/arm/mach-sa1100/h3600.c
@@ -247,7 +247,7 @@ static void __init h3xxx_map_io(void)
 
 	sa1100_register_uart_fns(&h3600_port_fns);
 	sa1100_register_uart(0, 3); /* Common serial port */
-//	sa1100_register_uart(1, 1); /* Microcontroller on 3100/3600 */
+	sa1100_register_uart(1, 1); /* Microcontroller on 3100/3600 */
 
 	/* Ensure those pins are outputs and driving low  */
 	PPDR |= PPC_TXD4 | PPC_SCLK | PPC_SFRM;
-- 
1.5.6.4

